### ISSUE TITLE
 Standard names: rename water_flux_correction

### ISSUE THREAD

**Issue by taylor13** [cf-convention/discuss#205]
In the description of water_flux_correction, it states "A positive flux correction is downward i.e. added to the ocean."  This is clear but the direction of other fluxes is indicated in the standard_name itself (using terms such as "upward", "upwelling", "downward", etc.).  Other CF names take the more explicit form ``water_flux_into_sea_water_due_to_X`` (e.g., ``water_flux_into_sea_water_due_to_surface_drainage``) or ``water_flux_into_sea_water_from_X`` (e.g., ``water_flux_into_sea_water_from_land_ice``). 

For clarity I suggest the current standard_name (``water_flux_correction``) be made an alias for a new standard_name: ``water_flux_into_sea_water_due_to_correction``.  

(This variable is used in the CMIP data archive, so there is a use case calling for improving clarity.)

**Comment by JonathanGregory** [comment]
Although it repeats "flux" I think `water_flux_into_sea_water_due_to_flux_correction` would be clearer, because "flux correction" is the name of the method, while "correction" is generic. Avoiding repetition, it could be "flux adjustment", which equally has been used. There is also a standard name `heat_flux_correction`. Should it be similar treated?

**Comment by taylor13** [comment]
I'd be happy with any of the proposed options, but to my ear (as trained by past usage) I like best ``water_flux_into_sea_water_due_to_flux_adjustment`` and ``heat_flux_into_sea_water_due_to_flux_adjustment`` .  The old names would become aliases, right?

**Comment by JonathanGregory** [comment]
I agree with your preferences, Karl. Yes, the old names would become aliases.

**Comment by feggleton** [comment]
Hi @JonathanGregory @taylor13, I'm happy with this clarification change to both existing names, which will create aliases. 

I have added this common phrase to the description: The specification of a physical process by the phrase "due_to_" process means that the quantity named is a single term in a sum of terms which together compose the general quantity named by omitting the phrase.

I have added to the editor and marked as accepted due to the time which has passed with no comment, this can then go into the next update. If you do not agree then let me know.
